app/assets/javascripts/jquery.infinite-pages.js.coffee

```diff
@@ -25,44 +25,44 @@ Released under the MIT License
       state:
         paused:  false
         loading: false
-    
+
     # Constructs the new InfinitePages object
     #
     # container - the element containing the infinite table and pagination links
     constructor: (container, options) ->
       @options = $.extend({}, @defaults, options)
-      @$container = $(container)
+      @container = '.'+container.className;
```

Comment:
This introduces new dependencies on the container argument. With your change it look like it has to be a DOM element with a class name instead of a CSS selector